src/Sulu/Component/Rest/ListBuilder/AbstractListBuilder.php

```diff
@@ -242,4 +254,12 @@ public function addGroupBy(AbstractFieldDescriptor $fieldDescriptor)
     {
         $this->groupByFields[$fieldDescriptor->getName()] = $fieldDescriptor;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function inArray(AbstractFieldDescriptor $fieldDescriptor, $values){
```

Comment:
Instead of having two different methods (`in` and `inArray`), I would prefer to put this null logic into the `in`-method. IMHO this difference is too small to create another function, which again has to be maintained in the future.